src/pip/_internal/wheel.py

```diff
@@ -715,7 +715,7 @@ def build(self, requirements, session, autobuilding=False):
                 pass
             elif autobuilding and req.link and not req.link.is_artifact:
                 # VCS checkout. Build wheel just for this run.
-                ephem_cache = True
+                buildset.append((req, True))
```

Comment:
Wouldn't it be better to move the `ephem_cache = False` initialization above to the code block using it bellow?

Reply:
It would. :)